### DISCUSSION TITLE
제페토 기본UI 숨기기 질문

### DISCUSSION THREAD

**Discussion by koldshine** [naverz/zepeto-studio-kor#936] [5. Q&A]
유니티로 월드 개발중인데요 제페토 기본UI(좌상단 버튼들)를 숨기고 싶습니다
같은 질문이 예전에도 올라와있던데
현재 이 기능이 추가되었는지 궁금합니다

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 홈버튼은 변경이 불가능하고 채팅 버튼은 추후에 커스터마이징 가능하도록 제공할 예정입니다.
https://github.com/naverz/zepeto-studio-kor/issues/905#issuecomment-1169539433
위 이슈에서 참고 부탁드립니다.
감사합니다.
